### PR TITLE
add mention of mesh annotation

### PR DESCRIPTION
**Pull request by devadvocado**

### CODE REVIEW

File: docs/docs/0.4.0/documentation/overview.md [hunk 1]
```diff
@@ -63,6 +63,22 @@ You can learn more about sidecar injection in the section on [Dataplanes](./dps-
 <img src="/images/docs/0.2.0/diagram-08.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
+### Specify Mesh for Services
```
Comment (jakubdyszkiewicz) [inline docs/docs/0.4.0/documentation/overview.md:66]: for Pods or Dataplanes, not for Services

Comment (jakubdyszkiewicz) [reply docs/docs/0.4.0/documentation/overview.md]: What I meant was `Specify Mesh for Pods` or ``Specify Mesh for Dataplanes` (I think second one is slightly better), not `Specify Mesh for Pods or Dataplanes`

**Comment by jakubdyszkiewicz** [comment]
Change the base for master

File: docs/docs/0.4.0/documentation/overview.md [hunk 2]
```diff
@@ -63,6 +63,22 @@ You can learn more about sidecar injection in the section on [Dataplanes](./dps-
 <img src="/images/docs/0.2.0/diagram-08.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
+### Specify Mesh for Pods or Dataplanes
+
+When deploying services in Kubernetes, you can determine which Mesh you want the service to be be in by using the `kuma.io/mesh: $MESH_NAME` annotation. This annotation would be applied to a deployment like so:
```
Comment (subnetmarco) [inline docs/docs/0.4.0/documentation/overview.md:68]: @devadvocado this is also possible to do in a `Namespace` correct? If yes, this description is not accurate.

Comment (jakubdyszkiewicz) [reply docs/docs/0.4.0/documentation/overview.md:68]: No, this is only possible on Deployment

**Pull request merged**